tx/models/qwen3.py

```diff
@@ -138,12 +143,7 @@ def __call__(
         hidden_states = self.mlp(hidden_states)
         hidden_states = residual + hidden_states
 
-        outputs = (hidden_states,)
-
-        if output_attentions:
-            outputs += (self_attn_weights,)
-        
-        return outputs
+        return hidden_states, self_attn_weights if output_attentions else None
```

Comment:
Nice simplification, I think we actually don't need this any more now that you adapted the attention layer to only return the attention weights if `output_attentions` is set. Instead, we can just do

```
return hidden_states, self_attn_weights
```

here :)